elpis/engines/common/output/elan.xsl

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
This file should go in elpis/engines/common/output/templates/elan.xml. You will have to create the templates folder.